docs/documentation/upgrading/topics/changes/changes-26_2_0.adoc

```diff
@@ -0,0 +1,15 @@
+== Breaking changes
+
+Breaking changes are identified as requiring changes from existing users to their configurations.
+
+=== Changes to port behaviour with the `X-Forwarded-Host` header
+
+The `X-Forwarded-Host` header can optionally also contain the port. In previous versions when the port was omitted from the header,
+{project_name} fell back to the actual request port. For example if {project_name} was listening on port 8080 and the request contained
+`X-Forwarded-Host: example.com` header, the resolved URL was `http://example.com:8080`.
```

Comment:
This link isn't supposed to be clicked, therefore don't render it as a link. Once this has changed, there is no need to add it to the `ignored-links` any more.

```suggestion
`X-Forwarded-Host: example.com` header, the resolved URL was `+http://example.com:8080+`.
```